src/index.js

```diff
@@ -120,6 +121,17 @@ export {
 } from './components/DatePicker/styles/DateRangePickerWrapper';
 export { default as DragBox } from './components/DragBox';
 export { default as DragBoxSelect } from './components/DragBox/DragBoxSelect';
+export { default as DragContainer } from './components/DragContainer';
+export { default as context } from './components/DragContainer/context';
```

Comment:
Yeah, this is what I was talking about. I think we should use a more descriptive name.

Reply:
That's fair. I would rather just not export it if that was possible. I don't love that this file just ignores all of the index files in its children – it would make more sense to just export those contents. That gives us more control over what actually gets exported by the library, whereas currently it appears we just export everything.

Reply:
Yeah, the system could use a revision. It has been convenient for exposing all the overrideable 'style' components though.